public/index.html

```diff
@@ -21,6 +21,7 @@
     id: "course-v1:edX+DemoX+Demo_Course",
     revision: "",
     base_url: "http://localhost:18010",
+    max_file_size_in_mbs: 10,
   },
```

Comment:
This means the release of this will be dependent on the need to pass this in from Studio in the studioContext, right?

Reply:
Yep! I'll need to make a PR to the platform to pass that in. It is [max_file_size_in_mbs](https://github.com/edx/edx-platform/blob/d0687e97b18a54d84c2534256f193a9ace35ab88/cms/templates/asset_index.html#L111).

Reply:
One more question as I look at this -- does this make sense to be in courseDetails? Mostly because it's not something that is course specific, or should we have another object for it?

Reply:
Right on! What about `upload_settings`?

Reply:
I updated this setting to be nested inside of `upload_settings`.